dist/stat_data.py

```diff
@@ -839,27 +839,27 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Cache content statistics
     ##########################################
```

Comment:
This might be the change we discussed last week, but up on line 260 we have the `Cache and eviction statistics` stats with lines like `    CacheStat('cache_eviction_app_fail', 'page evict failures by application threads'),` which should now be EvictStats